### ISSUE TITLE
12.0.2 - index 2 out of range for slice of length 1

### ISSUE THREAD

**Issue by hamptokr** [XAMPPRocky/tokei#568]
On the latest version I get `thread '<unnamed>' panicked at 'index 2 out of range for slice of length 1', ..userpath../tokei-12.0.2/src/utils/ext.rs:55:10`

**Comment by XAMPPRocky** [comment]
Thank you for your issue! @hamptokr Would you be able to provide link to the code, otherwise I don’t have a way to see what’s causing the problem

**Comment by jgilchrist** [comment]
@XAMPPRocky I was able to reproduce with the following HTML file:

```html
<html>
  <script></script>
</html>
```

While this works fine:

```html
<html>
  <script>
  </script>
</html>
```

**Comment by XAMPPRocky** [comment]
@jgilchrist Thanks for investigating, and verifying that my fix solved the problem!

**Comment by hamptokr** [comment]
I am still having this issue but it is in a large internal Vue project, trying to use -vvv to find the exact file its having trouble with.

**Comment by hamptokr** [comment]
Found the file, it is a dead file and I'll remove it but it might be useful to solve the issue. I think it's caused by the space between template tags.
```Vue
<template> </template>

<script lang="ts">
import Vue from 'vue'
export default Vue.extend({})
</script>

<style lang="scss" scoped></style>
```

**Comment by XAMPPRocky** [comment]
Thanks I had feeling that was it. I’ll post a new release with a fix for the issue tomorrow.

**Comment by XAMPPRocky** [comment]
This has been fixed in `12.0.4`.
